lib/ret_web/controllers/api/v1/scene_controller.ex

```diff
@@ -0,0 +1,29 @@
+defmodule RetWeb.Api.V1.SceneController do
+  use RetWeb, :controller
+
+  alias Ret.Scene
+  alias Ret.Repo
+
+  # Limit to 1 TPS
+  plug(RetWeb.Plugs.RateLimit)
```

Comment:
this probably should only be applied to the `create` -- i am not sure how to do it offhand